src/CISDSC/dscresources/CIS_Microsoft_Windows_10_Enterprise_Release_2004/CIS_Microsoft_Windows_10_Enterprise_Release_2004.schema.psm1

```diff
@@ -0,0 +1,4711 @@
+        [int32]$916LogFileSize = 16384,
+        [ValidateRange(16384,2147483647)]
+        [int32]$926LogFileSize = 16384,
+        [ValidateRange(16384,2147483647)]
+        [int32]$938LogFileSize = 16384
+    )
+
+    Import-DSCResource -ModuleName 'PSDesiredStateConfiguration'
+    Import-DSCResource -ModuleName 'AuditPolicyDSC' -ModuleVersion '1.4.0.0'
+    Import-DSCResource -ModuleName 'SecurityPolicyDSC' -ModuleVersion '2.10.0.0'
+
+    if($ExcludeList -notcontains '2.3.1.5' -and $PSBoundParameters.Keys -notcontains '2315AccountsRenameadministratoraccount'){
+        throw 'Please add "2.3.1.5" to the ExcludeList or provide a value for "2315AccountsRenameadministratoraccount"'
+    }
+    if($ExcludeList -notcontains '2.3.1.6' -and $PSBoundParameters.Keys -notcontains '2316AccountsRenameguestaccount'){
+        throw 'Please add "2.3.1.6" to the ExcludeList or provide a value for "2316AccountsRenameguestaccount"'
+    }
+    if($ExcludeList -notcontains '2.3.7.5' -and $PSBoundParameters.Keys -notcontains '2375LegalNoticeText'){
+        throw 'Please add "2.3.7.5" to the ExcludeList or provide a value for "2375LegalNoticeText"'
+    }
+    if($ExcludeList -notcontains '2.3.7.6' -and $PSBoundParameters.Keys -notcontains '2376LegalNoticeCaption'){
+        throw 'Please add "2.3.7.6" to the ExcludeList or provide a value for "2376LegalNoticeCaption"'
+    }
+
+    if($ExcludeList -notcontains '1.1.1' -and $LevelOne){
+        AccountPolicy "1.1.1 - (L1) Ensure Enforce password history is set to 24 or more password(s)" {
+            Enforce_password_history = 24
+            Name = 'Enforce_password_history'
+        }
+    }
+    if($ExcludeList -notcontains '1.1.2' -and $LevelOne){
+        AccountPolicy "1.1.2 - (L1) Ensure Maximum password age is set to 60 or fewer days but not 0" {
+            Maximum_Password_Age = $112MaximumPasswordAge
+            Name = 'Maximum_Password_Age'
+        }
+    }
+    if($ExcludeList -notcontains '1.1.3' -and $LevelOne){
+        AccountPolicy "1.1.3 - (L1) Ensure Minimum password age is set to 1 or more day(s)" {
+            Minimum_Password_Age = $113MinimumPasswordAge
+            Name = 'Minimum_Password_Age'
+        }
+    }
+    if($ExcludeList -notcontains '1.1.4' -and $LevelOne){
+        AccountPolicy "1.1.4 - (L1) Ensure Minimum password length is set to 14 or more character(s)" {
+            Minimum_Password_Length = 14
+            Name = 'Minimum_Password_Length'
+        }
+    }
+    if($ExcludeList -notcontains '1.1.5' -and $LevelOne){
+        AccountPolicy "1.1.5 - (L1) Ensure Password must meet complexity requirements is set to Enabled" {
+            Name = 'Password_must_meet_complexity_requirements'
+            Password_must_meet_complexity_requirements = 'Enabled'
+        }
+    }
+    if($ExcludeList -notcontains '1.1.6' -and $LevelOne){
+        Registry "1.1.6 - (L1) Ensure Relax minimum password length limits is set to Enabled" {
```

Comment:
Is this right? I'm not used to 'Relax' and 'Enabled' being the expected setting.

Reply:
according to CIS it is
![image](https://user-images.githubusercontent.com/28571284/93239753-d4e98080-f748-11ea-9ede-808407905daf.png)